app/models/talentlms/facade.rb

```diff
@@ -49,6 +49,19 @@ def create_account(user)
       result['login_key']
     end
 
+    # Enroll a user in a course in TalentLMS
+    #
+    # @param user [User] the user
+    # @param course_id [Integer] the id of the course
+    def enroll(user, course_id)
+      login = Login.find_by(user: user)
+      return false if login.nil?
+
+      @api.post('addusertocourse', {course_id: course_id, user_id: login.lms_user_id})
+    rescue RuntimeError => e
+      raise e unless e.message == 'The requested user is already enrolled in this course'
```

Comment:
Just to be less brittle, I might change this to `e.message.include?('already enrolled')`